src/imports/logger.js

```diff
@@ -52,8 +56,8 @@ const logger = winston.createLogger({
     timestamp(),
     format.errors({ stack: true }),
     printf(({ level, timestamp, from, userId, ...rest }) => {
-      const logPayload = mapKeys(rest, (_, key) => (key === SPLAT ? 'context' : key))
-      const stringifiedPayload = JSON.stringify(logPayload, (_, logValue) => formatLogValue(logValue))
+      const context = rest[SPLAT]
```

Comment:
so, [SPLAT] proparty should be in the payload, we have to **copy** it to context not just rename ?